From 4c90fc973ce1d77e4cbb0b989ae4e4db6476a803 Mon Sep 17 00:00:00 2001
From: Daniel Emminizer <daniel.m.emminizer.civ@us.navy.mil>
Date: Tue, 29 Apr 2025 10:45:14 +0000
Subject: [PATCH 25/29] SDK: Fix Invalid Return Statement

**Description:** Removed invalid return from void function, identified by Klocwork.

**Testing Performed:** Build on Linux.
---
 SDK/simData/DataTypes.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SDK/simData/DataTypes.h b/SDK/simData/DataTypes.h
index 99a00fd2..1e89e3e5 100644
--- a/SDK/simData/DataTypes.h
+++ b/SDK/simData/DataTypes.h
@@ -506,7 +506,7 @@ namespace simData
     /// Returns the requested entry;
     const Entry& entry(int index) const { return entries_[index]; }
     /// Remove all entries
-    void clear_entry() { return entries_.clear(); }
+    void clear_entry() { entries_.clear(); }
     /// Add an entry
     Entry* add_entry() { entries_.push_back(Entry()); return &entries_.back(); }
 
-- 
2.47.1.windows.2

